.eslintrc.json

```diff
@@ -28,6 +28,15 @@
   "rules": {
     "react/react-in-jsx-scope": "off",
     "react/prop-types": "off",
+    "jsx-a11y/label-has-associated-control": [
+      2,
+      {
+        "labelComponents": ["CustomInputLabel"],
+        "labelAttributes": ["label"],
+        "controlComponents": ["CustomInput"],
```

Comment:
_"`controlComponents` is a list of custom React Components names that will output an input element."_

We don't have a component named "CustomInput". We do have a "Switch" though. And maybe other components like "TextInput" as well, but feel free to consider them outside the scope of this PR.